apps/beeai-sdk/tests/e2e/test_runs.py

```diff
@@ -147,27 +147,6 @@ async def test_run_cancel_awaiter(awaiter: tuple[Server, Client]) -> None:
     assert task_response.status.state == TaskState.canceled
 
 
-async def test_run_cancel_slow_echo(slow_echo: tuple[Server, Client]) -> None:
```

Comment:
why is the test invalid?

Reply:
It waits for task completion and tries to cancel it after.